docs/pages/connect-your-client/gui-clients.mdx

```diff
@@ -317,8 +344,16 @@ Provide your Redis username as `Username` and password as `Password`.
 
 ![Redis Insight Configuration](../../img/database-access/guides/redis/redisinsight-add-config.png)
 
-Next, check the `Use TLS` and `Verify TLS Certificates` boxes and copy the CA certificate returned by `tsh proxy db`.
-Copy the private key and certificate to corresponding fields.
+Next, check the `Use TLS` and `Verify TLS Certificates` boxes. Copy the files at
```

Comment:
```suggestion
Next, check the `Use TLS` and `Verify TLS Certificates` boxes. Copy the contents of the files at
```

(Copying the contents of the certs is kinda cursed from UX standpoint, I bet most people won't even realize those are text files.)